app/assets/javascripts/application.js.erb

```diff
@@ -463,35 +317,84 @@ const removeExtraInputs = () => {
 
 class _LobstersFunction {
   constructor () {
-    this.curUser = 'will';  //done
-  }
+    this.curUser = 'will';  // remove line when done with production
```

Comment:
As I [mentioned the other day](https://github.com/lobsters/lobsters/pull/1098#issuecomment-1160486634) this should move to a `data` attribute you can read. I've updated the server side (bbd4887) to present this, so please update this PR to read that. But switching over all the references from `Lobsters.curUser` to `Lobster.curUser` can wait for your next PR.